app/views/trustee/AddATrusteeView.scala.html

```diff
@@ -31,31 +38,33 @@
 
 @main_template(
     title = s"${errorPrefix(form)}$heading"
-    ) {
+) {
 
     @formHelper(action = AddATrusteeController.submitAnother(), 'autoComplete -> "off") {
 
-        @components.error_summary(form.errors)
+        @errorSummary(form.errors, AddATrustee.options)
 
-        @components.heading(heading)
+        @headingComp(heading)
 
-        @components.data_list(inProgressTrustees, "inprogress", "trustee")
+        @addToList(inProgressTrustees, "inprogress", "trustee")
 
-        @components.data_list(completeTrustees, "complete", "trustee")
+        @addToList(completeTrustees, "complete", "trustee")
 
         @if(!isLeadTrusteeDefined) {
-            @components.inset_text("addATrustee.lead-trustee.required", "addATrustee")
+            <div id="inset-text--addATrustee" class="indent">
```

Comment:
the `class="indent"` should be replaced with ` class="govuk-inset-text"` I know it's v rare that LT is not defined but still

it's here and on MaxedOutTrustees View

Reply:
Updated